guides/common/modules/ref_system-requirements.adoc

```diff
@@ -43,14 +50,16 @@ endif::[]
 
 {Project} only supports `UTF-8` encoding.
 If your territory is USA and your language is English, set `en_US.utf-8` as the system-wide locale settings.
-For more information about configuring system locale in {EL}, see https://access.redhat.com/documentation/en-us/red_hat_enterprise_linux/8/html/configuring_basic_system_settings/assembly_changing-basic-environment-settings_configuring-basic-system-settings#proc_configuring-the-system-locale_assembly_changing-basic-environment-settings[Configuring System Locale guide].
+ifdef::foreman-el,katello,satellite[]
+For more information about configuring system locale in {EL}, see https://access.redhat.com/documentation/en-us/red_hat_enterprise_linux/8/html/configuring_basic_system_settings/assembly_changing-basic-environment-settings_configuring-basic-system-settings#proc_configuring-the-system-locale_assembly_changing-basic-environment-settings[Configuring the system locale] in _{RHEL} 8 {Configuring basic system settings}_.
+endif::[]
 
 {ProjectServer} and {SmartProxyServer} do not support shortnames in the hostnames.
 When using custom certificates, the Common Name (CN) of the custom certificate must be a fully qualified domain name (FQDN) instead of a shortname.
 This does not apply to the clients of a {Project}.
 
 ifdef::satellite[]
-* All {ProjectServer}s must be on the same version.
+* If you deploy multiple {ProjectServer}s, they must be on the same OS version.
```

Comment:
@ekohl Is it accurate that the version in question is of the OS? Or is supposed to be the Sat version?

Reply:
I think this is an interesting one. It was added in https://github.com/theforeman/foreman-documentation/pull/1293 but I wonder if it's really true. I mean, AFAIK Satellite supports an upgrade path where you start to migrate your Foreman server from EL7 to EL8 but keep your content proxies on EL7 and migrate as time permits. Additionally, there's n-1 support where the Satellite is on x.y and the Capsules are on x.(y-1) so I wonder if where it came from.

Reply:
Okay, so this comes from ISS Export/Import and it's about Sat versions:
> In a disconnected enviro, if some Satellite servers are on 6.x version and others are on 6.10, the content import and export does not work. The requirement is all Satellite servers should be on the same version.

I'll rephrase it.